src/main/kotlin/io/openfuture/state/controller/domain/dto/WalletDto.kt

```diff
@@ -0,0 +1,10 @@
+package io.openfuture.state.controller.domain.dto
```

Comment:
1. Wrong package name. -> `io.openfuture.controller.dto`. Domain and DTO are incompatible. 
1. I would recommend defining DTO classes as well as Request/Response wrappers in the controllers that are using them. For sure, if the DTO is wide-spread it doesn't make sense. However, in most cases, DTO needs to be defined within a specific controller that handles it.

Reply:
So, Define that `WalletDto` as an inner static class of `WalletController`. Because there is the only place when we are using it so fat.